### ISSUE TITLE
[Global setting] Focus to follow corner radius setting

### ISSUE THREAD

**Issue by melissaperreault** [Shopify/dawn#1018]
### Expectations

- [ ] Let's have the focus [keep the same roundness/corner radius](https://screenshot.click/21-12-5dp11-3u9ew.png) of the element it is surrounding.
- [ ] Sort by gets roundness on focus [but Filter doesn’t](https://screenshot.click/21-12-35csg-pr2up.png).

**Comment by katycobb** [comment]
Hi @melissaperreault  - would this issue cover what I'm observing here? Not sure if it needs a new bug. This appears when clicking the "Share" icon on the PDP.
<img width="549" alt="06-34-gi6t9-xwuok" src="https://user-images.githubusercontent.com/93404168/148433494-fc8fec87-3534-4c4a-9b23-d74a6ff9a3f1.png">

**Comment by melissaperreault** [comment]
> would this issue cover what I'm observing here

@katycobb Yes, this issue will fix! I'll be pairing with Sofia to make this more appealing! 😊
